app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dimen_80dp">80dp</dimen>
```

Comment:
Not very well idea extracted like this.
For different kind(mergin, padding, textSize etc.) of values, must be had different resources.
base_margin>5dp<
base_padding>5dp<